src/Layers/FeatureLayer/FeatureLayer.js

```diff
@@ -90,8 +103,20 @@ EsriLeaflet.Layers.FeatureLayer = EsriLeaflet.Layers.FeatureManager.extend({
         return;
       }
 
+      // polylines and polygons
       if (layer && layer.setLatLngs) {
-        this._updateLayerGeometry(layer, geojson);
+        this._updateLayer(layer, geojson);
+        // to do: make sure appropriate style is set
+        // this.resetStyle();  ?
+        return;
+      }
+
+      // points
+      if (layer && layer.setLatLng) {
+        this._updateLayer(layer, geojson);
+        var iconTest = this.options.pointToLayer(geojson, L.latLng(geojson.geometry.coordinates[1], geojson.geometry.coordinates[0]));
+        var updatedIcon = iconTest.options.icon;
+        layer.setIcon(updatedIcon);
```

Comment:
That is easier to follow.  Is it possible to have a pointToLayer function and no icon?  It doesn't seem easy to do, just wondering.

I guess what I'm wondering the most is what scenarios might occur that we should be setting the icon back to the default leaflet icon.